doc/index.rst

```diff
@@ -277,11 +277,12 @@
             <div class="span4">
                 <h4>Community</h4>
                 <ul>
-                <li><em>About us</em> See <a href="about.html">authors</a> # scikit-learn</li>
+                <li><em>About us</em> See <a href="about.html#people">authors</a></li>
                 <li><em>More Machine Learning</em> Find <a href="related_projects.html">related projects</a></li>
-                <li><em>Questions?</em> See <a href="http://stackoverflow.com/questions/tagged/scikit-learn">stackoverflow</a> # scikit-learn</li>
+                <li><em>Questions?</em> See <a href="faq/">FAQ</a> and <a href="http://stackoverflow.com/questions/tagged/scikit-learn">stackoverflow</a> #scikit-learn</li>
```

Comment:
I'd like to keep this line short. Could you remove "#scikit-learn" in the text, and use the URL:

http://stackoverflow.com/questions/tagged/scikit-learn